lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -45,7 +45,8 @@ function contentDescription(content) {
     case 'vitalsource':
       return 'Book from VitalSource';
     default:
-      return 'Other content';
+      /* istanbul ignore next */
```

Comment:
I presume this is an in-development/prototype path so doesn't need coverage?

Reply:
This code path existed only to convince TypeScript that the function always returns a string, if it returns a value. Throwing an error is a more obvious way to make it clear that the code path should never be executed.